olivertwist/rules/no_references_outside_of_its_staging_area.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+"""Staging models should have a single source.
+
+Copyright (C) 2020, Auto Trader UK
+Created 22. Dec 2020 10:29
+
+"""
+
+from typing import List, Tuple
+
+from olivertwist.manifest import Manifest, Node
+from olivertwist.ruleengine.rule import rule
+from olivertwist.rules.utils import partition
+
+
+@rule(id="no-references-outside-of-its-staging-area", name="No references outside of its staging area")
```

Comment:
Is it clearer if we name the rule `no-references-outside-of-own-staging-area`?

Reply:
how can you tell that i struggled with that? :)